misc/python/materialize/cloudtest/wait.py

```diff
@@ -9,13 +9,19 @@
 
 
 import subprocess
+from typing import Optional
 
 from materialize import ui
 from materialize.ui import UIError
 
 
 def wait(
-    condition: str, resource: str, timeout_secs: int = 300, context: str = "kind-kind"
+    condition: str,
+    resource: str,
+    timeout_secs: int = 300,
+    context: str = "kind-kind",
+    *,
```

Comment:
TIL!

Reply:
One of the lesser known features introduced with Python 3! I really like keyword-only args for cases like this.